src/control/cmd/dmg/pretty/system.go

```diff
@@ -180,3 +180,50 @@ func PrintSystemStartResponse(out, outErr io.Writer, resp *control.SystemStartRe
 func PrintSystemStopResponse(out, outErr io.Writer, resp *control.SystemStopResp) error {
 	return printSystemResults(out, outErr, resp.Results, &resp.AbsentHosts, &resp.AbsentRanks)
 }
+
+func printSystemCleanupRespVerbose(out io.Writer, resp *control.SystemCleanupResp) error {
+	if len(resp.Results) == 0 {
+		fmt.Fprintln(out, "no handles cleaned up")
+		return nil
+	}
+
+	titles := []string{"Id", "Handles Revoked"}
+	formatter := txtfmt.NewTableFormatter(titles...)
+
+	var table []txtfmt.TableRow
+	for _, r := range resp.Results {
+		row := txtfmt.TableRow{
+			"Id":              r.PoolID,
```

Comment:
Might be better to have this label say "Pool" since now it is always a pool.

Reply:
Ok changed it.